src/sentry/models/grouphash.py

```diff
@@ -35,3 +36,5 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_grouphash"
         unique_together = (("project", "hash"),)
+
+    __repr__ = sane_repr("group_id")
```

Comment:
wouldn't we want the group hash in the repr as well?

Reply:
Yeah, I suppose it depends on what kind of debugging you’re doing. In the context of tests (which is how I came to make the change), it's much easier to check the 1- or 2-digit ids against each other than the 32-character hashes. But you're right, if this were in prod (where the ids are also long and hard to compare at a glance), the hashes would probably be the better thing to use. I can add it.